internal/extramanifest/extramanifest.go

```diff
@@ -18,28 +18,55 @@ package extramanifest
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"os"
 	"path/filepath"
 	"strconv"
 
 	"github.com/go-logr/logr"
 	lcav1alpha1 "github.com/openshift-kni/lifecycle-agent/api/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
+	"github.com/openshift-kni/lifecycle-agent/internal/common"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/yaml"
 )
 
-const extraManifestPath = "extra-manifests/"
+const ExtraManifestPath = "/opt/extra-manifests"
```

Comment:
It may not be suitable to have this in the /opt/openshift folder since it is deleted once the systemd service completes its task. However, these manifests are applied at a later stage, after the cluster has achieved stability, by the LCA.
These manifests are user provided through the `spec.extraManifests` in IBU CR. Unlike the manifests applied in post-pivot.go, they serve distinct purposes and are applied at different times. From my perspective, I don't anticipate any issues as long as proper cleanup is done once the task is completed.